src/qrcode/QRTabBar.tsx

```diff
@@ -91,9 +83,17 @@ export default function QRTabBar({
         <TopBarIconButton
           icon={
             closeIcon === CloseIcon.BackChevron ? (
-              <BackChevron color={color} />
+              // BackChevron doesn't support Animated color
+              <BackChevron color={defaultColor} />
             ) : (
-              <Times color={color} />
+              // using `animatedColor` with animated svg causes android crash since
+              // upgrading react-native-svg to v13. there are some suggested solutions
+              // linked below, but none that i could get to work after many attempts. since
+              // this is a relatively low impact feature, i'm going to leave it as is for
+              // now.
+              // https://github.com/software-mansion/react-native-svg/issues/1976
+              // https://github.com/software-mansion/react-native-reanimated/issues/3775
+              <Times color={Platform.OS === 'ios' ? animatedColor : defaultColor} />
```

Comment:
So, on iOS, the animated color doesn't work for `BackChevron`, but works for `Times`?

Reply:
yes